integration-tests/sqoop-parquet-full-load/tables.yml

```diff
@@ -9,7 +9,7 @@ impala_cmd: "{{ impala_cmd }}"
 source_database:
   name: "integration_test" # Source database name
   connection_string: {{ connection_string }} # Source database connection string. Should be kept in 'env.yml'
-  cmd: "mysql -P 3306 -uroot -ppipewrench -h mysql <"
+  cmd: "sqoop eval --connect {{ connection_string }} --username {{ source_db_user_name }} --password pipewrench --query " #"mysql -P 3306 -uroot -ppipewrench -h mysql <"
```

Comment:
Do we still need the commented version? Is this just so we don't have to rely on the mysql client?

Reply:
No, we do not need the commented out version.  We needed to switch to `sqoop eval` as the docker container does not have the `mysql` cli tools.